packages/perspective-jupyterlab/src/config/plugin.config.js

```diff
@@ -25,11 +25,45 @@ module.exports = {
     plugins: [new webpack.ContextReplacementPlugin(/moment[\/\\]locale$/, /(en|es|fr)$/), new PerspectivePlugin()],
     module: {
         rules: [
+            {
+                test: /\.less$/,
+                exclude: /themes/,
+                use: [
+                    {
+                        loader: "css-loader"
+                    },
+                    {
+                        loader: "clean-css-loader",
+                        options: {
+                            level: 2
+                        }
+                    },
+                    {
+                        loader: "less-loader"
+                    }
+                ]
+            },
+            {
```

Comment:
Likewise, removing this will not mix well with traditional use of `html-loader` - these are for strings in the shadow DOM.

Reply:
I think this is no longer needed with my changes to `perspective-viewer` and the plugins to publish JS files only.

Reply:
I was correct, these are no longer needed in this package so I've removed them.